code/game/machinery/computer/robot.dm

```diff
@@ -88,21 +91,38 @@
 			if(allowed(usr))
 				var/mob/living/silicon/robot/R = locate(params["ref"]) in GLOB.silicon_mobs
 				if(can_control(usr, R) && !..())
-					if(isAI(usr) && (R.ai_lockdown && R.lockcharge || !R.lockcharge) || !isAI(usr))
-						R.ai_lockdown = FALSE
-						if(isAI(usr) && !R.lockcharge)
+					if(isAI(usr))
+						if(R.ai_lockdown && R.lockcharge)
+							R.ai_lockdown = FALSE
+							src.lock_unlock_borg(R)
```

Comment:
```suggestion
							lock_unlock_borg(R)
```
`src.` is implicit most of the time, and I don't think there is any ambiguity here?